src/components/DisplayExplanation.vue

```diff
@@ -19,8 +19,12 @@
         :blackboardRef="selfRef"
         ref="DoodleVideo"
         :blackboardId="explanation.id"
+        :thumbnail="explanation.thumbnail"
         :audioUrl="explanation.audioUrl"
         :hasBetaOverlay="true"
+        @click="handleVideoClick()"
```

Comment:
I think the DoodleVideo can completely handle it's own click behavior - that is, on clicking the play overlay, it will fetch all the resources it can fetch, then play everything it can play. I believe this will not hinder flexibility, and will lead to better encapsulation and simplicity. 

Home: DoodleVideos automatically fetches and plays (because no thumbnail prop is provided), but if the user wants to click play again, the DoodleVideo will fetch resources (if not yet fetched), then play. All the meanwhile, the parent doesn't have to know anything. 
DisplayExplanation: DoodleVideo works as intended.